Would Docker image need anything else installed?

Hi,

I tried to run API-ML locally with Docker image provided, but getting following error(and followed by more errors too). I thought if I use docker image I won't have to add/install any extra packages...should I still do anything extra to make it run?
```
docker run --rm --name api-layer --net host -d jbalhar/api-layer-development:1.0.1 /sbin/my_init
docker exec api-layer bash _run

```

```

> Task :api-catalog-ui:bundle

> frontend@0.3.0 build /var/src/api-layer/api-catalog-ui/frontend
> react-scripts build

Creating an optimized production build...
Failed to compile.npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! frontend@0.3.0 build: `react-scripts build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the frontend@0.3.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.


./src/components/Swagger/Swagger.jsx
Cannot find module: 'swagger-ui-react/swagger-ui'. Make sure this package is installed.

You can install this package by running: npm install swagger-ui-react/swagger-ui.




```

--- a/docker/README.md
+++ b/docker/README.md
@@ -43,7 +43,7 @@ The main goal of the image is to simplify running of all the types of the tests.
 To download and run the container it is necessary to install the [Docker Engine](https://www.docker.com/) first. The following commands start new instance of the container and log you into the bash with root privileges. The last line then updates the version of the api-layer and run the unit tests, integration tests and end to end tests. 
 
 ```
-docker run --rm --name api-layer -d jbalhar/api-layer-development:1.0.1 /sbin/my_init
+docker run --rm --name api-layer -d jbalhar/api-layer-development:1.13.1 /sbin/my_init
 docker exec api-layer bash _test
 ```
 

--- a/docker/development/README.md
+++ b/docker/development/README.md
@@ -1,0 +1,8 @@
+# Release new version
+
+With each release of the Zowe new version of this container needs to be published. The prerequisity to release is a full Docker instalation on the machine running the comands.
+
+1) Based on the OS open the PowerShell or Bash 
+2) Change working directory to api-layer/docker/development
+3) docker build --no-cache -t jbalhar/api-layer-development:{versionOfZowe} .
+4) docker push jbalhar/api-layer-development:{versionOfZowe}

--- a/docker/development/_run
+++ b/docker/development/_run
@@ -1,12 +1,20 @@
+#!/bin/bash
 export NVM_DIR="$HOME/.nvm" \
     && [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  \
     && [ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  \
     && nvm use v12.16.1
 
-# Update and build
+# Update
 cd /var/src/api-layer
 git fetch
 git pull
+
+# Install dependencies
+cd /var/src/api-layer/api-catalog-ui/frontend
+pnpm install
+
+# Build
+cd /var/src/api-layer
 ./gradlew clean build
 
 # Run integration tests and local instances of the services

